test/images/sample-device-plugin/VERSION

```diff
@@ -1 +1 @@
-1.4
+1.5
```

Comment:
let's fix the commit message:
```
Update the sample device plugin to enable the e2e node tests (or any other entity with full access to the node filesystem) to control the registration process. We add a new environment variable `REGISTER_CONTROL_FILE`.
The value of this variable must be a file which prevents the plugin to register itself while it's present. Once removed, the plugin will go on and complete the registration. The plugin will automatically detect the parent directory on which the file resides and detect deletions, unblocking the registration process. If the file is specified but unaccessible, the plugin will fail. If the file is not specified, the registration process will progress as usual and never pause. The plugin will need read access to the parent directory.

This feature is useful because it is not possible to control the order in which the pods are recovered after node reboot/kubelet restart.

In this approach, the testing environment will create a directory and then a empty file to pause the registration process of the plugin. Once pointed to that file, the plugin will start and wait for it to be deleted. Only after the directory has been deleted, the plugin would proceed to registration.

In the subsequent...
```

Reply:
Thanks, will update now.